smart_open/tests/test_hdfs.py

```diff
@@ -30,24 +31,30 @@
 # Since these tests use cat, they will not work in an environment without cat,
 # such as Windows.
 #
+def cat(path):
+    executable = 'cat'
+    shell = False
+    if sys.platform == 'win32':
+        executable = 'type'
+        shell = 'True'
```

Comment:
This should be a bool.

```suggestion
        shell = True
```